linera-core/src/worker.rs

```diff
@@ -1120,6 +1135,11 @@ where
             NUM_ROUNDS_IN_CERTIFICATE
                 .with_label_values(&[log_str])
                 .observe(round.0 as f64);
+            if is_confirmed {
```

Comment:
Now we don't need `is_confirmed`, and can just check whether `confirmed_transactions > 0` here. (If that's necessary at all; not sure if `inc_by(0)` is any performance concern?)

Reply:
Ah right, sounds good. Shouldn't be, but might as well be safe